app/helpers/external/mailbluster_helper.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module External
```

Comment:
This suggests the location is too close to `Application`.  I would try app/lib or just lib/ and you can probably just have a `MailBluster` or similar class.